python/pyspark/sql/tests/udf_type_tests/README.md

```diff
@@ -0,0 +1,5 @@
+These tests capture input/output type interfaces between python udfs and the engine.
```

Comment:
Do we want to run this in CI? If so, we should add `__init__.py` ,and add this modul into `dev/sparktestsupport/modules.py`

Reply:
yes, let me try to add it. Do we need any other reviewer for the infra changes?